src/Internal/Process/HydraNode.purs

```diff
@@ -150,14 +155,30 @@ spawnHydraNode params handlers = liftEffect do
   option :: String -> String -> Array String
   option name val = [ "--" <> name, val ]
 
-  networkArgs :: Array String
+  optionMaybe :: forall (a :: Type). String -> (a -> String) -> Maybe a -> Array String
```

Comment:
how about:
```suggestion
  optionMaybe :: forall (a :: Type). String -> Maybe String -> Array String
```
then we can later use a `<$>` to map over Maybe a to get a Maybe String if needed